analyses/telomerase-activity-prediction/RUN-telomerase-activity-prediction.sh

```diff
@@ -0,0 +1,22 @@
+#enviroment settings
+set -e
+set -o pipefail
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+script_directory="$(perl -e 'use File::Basename;
+  use Cwd "abs_path";
+  print dirname(abs_path(@ARGV[0]));' -- "$0")"
+cd "$script_directory" || exit
+
+#generate collapsed data for count files 
+Rscript ../collapse-rnaseq/01-summarize_matrices.R -i ../../data/pbta-gene-counts-rsem-expected_count.stranded.rds -g ../../data/gencode.v27.primary_assembly.annotation.gtf.gz -m ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.stranded.rds -t ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed_table.stranded.rds
+Rscript ../collapse-rnaseq/01-summarize_matrices.R -i ../../data/pbta-gene-counts-rsem-expected_count.polya.rds -g ../../data/gencode.v27.primary_assembly.annotation.gtf.gz -m ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.polya.rds -t ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed_table.polya.rds
+
+mkdir results
```

Comment:
The failure in CI is related to this step and specifically to the lack of `-p`.
```suggestion
mkdir -p  results
```

`-p` makes it such that no error is thrown if the directory already exists ([`mkdir` man page](https://linux.die.net/man/1/mkdir)). The reason the `results` directory exists on the machine running CI is because you have now committed the EXTEND scores to the repository (thank you) in that directory. The first step in CI is basically to checkout your branch of the repository and that's why the directory already exists.

Reply:
We have removed the files as required. And also have added -p in the shell script. 
For the 2nd step do we need to wait till the first pull request is merged??